README.md

```diff
@@ -4,115 +4,72 @@
 
 ## Overview
 
-Use this [helm chart](https://helm.sh/docs/topics/charts/) to install IllumiDesk on AWS EKS. This chart depends on the [jupyterhub](https://zero-to-jupyterhub.readthedocs.io/en/latest/).
+Use this [helm chart](https://helm.sh/docs/topics/charts/) to install IllumiDesk into your Cluster. This chart depends on the [jupyterhub](https://zero-to-jupyterhub.readthedocs.io/en/latest/).
 
 This setup pulls images defined in the `illumidesk/values.yaml` file from `DockerHub`. To push new versions of these images or to change the image's tag(s) (useful for testing), then follow the instructions in the [build images section](#build-images).  
 
 ## Requirements
 
 - [helm >= v3](https://github.com/kubernetes/helm)
-- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
-- [Amazon EKS vended KUBECTL](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html)
-- [EKSCTL](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html)
+- [Standard Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- (Optional)[AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+- (Optional)[Amazon EKS vended KUBECTL](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html)
```

Comment:
I don't think the AWS vended `kubectl` is even worth mentioning. What we need to do is specify what version(s) of Kubernetes this chart has been tested with.